demos/python_demos/image_retrieval_demo/main.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
```

Comment:
For consistency with other demos, the main script should be called `image_retrieval_demo.py`.

Reply:
done